src/session.js

```diff
@@ -5,7 +5,7 @@ let sessions = {};
 function hydrate(request, sessionId) {
   let session = sessions[sessionId];
 
-  Object.assign(request, session, { sessionId });
+  Object.assign(request, { session, sessionId });
```

Comment:
Yes, instead of randomly placing session variables on the request, organize them into a session object on the request.